### PR TITLE
Add a bio for @githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher as requested in #4.

@githubstudent, what do you think of these changes?  I would :heart: your :eyes: and :thought_balloon:s on this. 
## To Do
- [ ] Be more specific about location
- [ ] Update @githubteacher name

### CODE REVIEW

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub Teacher
+
+**Name:** GitHub Teacher
```
Comment (jphelps92) [inline docs/githubteacher.md]: Is this on your birth certificate?

**Comment by githubteacher** [comment]
I really like these changes, but there are a few tweaks I would make.  I have made some inline comments.

File: docs/githubteacher.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+**Occupation:** Teacher
+
+**Hometown:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: There are a lot of San Franciscos in the :earth_africa:.  Maybe we should be more specific here.

File: docs/githubteacher.md (hunk 1 above)
Comment (githubteacher) [inline docs/githubteacher.md]: Yes. :smiley_cat:

**Comment by githubteacher** [comment]
I am referencing an earlier comment [here](https://github.com/githubteacher/developers-09-24-2015/pull/34#discussion_r40335363).

https://github.com/githubteacher/developers-09-24-2015/pull/34#discussion_r40335363

**Comment by githubteacher** [comment]
:+1:

**Comment by githubteacher** [comment]
:ship: it!

:shipit:

**Pull request merged**